packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -62,6 +63,44 @@ const propTypes = {
     ),
   ),
   /**
+   * An array of items to be displayed as help menu content.
+   */
+  helpItems: PropTypes.arrayOf(PropTypes.shape(
```

Comment:
This seems spread out all over the components. Really you shouldn't have to declare the shape here if its just passing it along into sub components. Let them declare the shape and throw their prop type mismatch warnings